src/mudlet.pro

```diff
@@ -230,10 +308,110 @@ macx:LIBS += \
 # will be used.
 DEFINES += LUA_DEFAULT_PATH=\\\"$${LUA_DEFAULT_DIR}\\\"
 
-# Enable the built-in updater by default. Linux packagers will find it useful to disable it
-# since package managers are responsible for updates there
-DEFINES += INCLUDE_UPDATER
 
+####################### Git Submodules check and install #######################
+# The "exists" tests and "include" directives uses qmakes internal path handling
+# (always uses '/' dirextroy separators); the git operations need to be done
```

Comment:
Typo

Reply:
Noted, but I'll leave it unless a further commit is necessary for other reasons.